presto-docs/src/main/sphinx/functions.rst

```diff
@@ -42,3 +42,4 @@ If you are looking for more information about a specific function or operator,
     functions/session
     functions/teradata
     functions/list
+    functions/ml
```

Comment:
I suppose `functions/list` should be bottom. Could you move this line to before `functions/list`?
Otherwise, the generated page will be like this.
<img width="297" alt="Screen Shot 2020-05-15 at 22 17 43" src="https://user-images.githubusercontent.com/6237050/82054478-f9314400-96f9-11ea-9a60-73b9b69febba.png">

Reply:
yes. Move up please